tools/telemetry/www/typings/mustache/mustache.d.ts

```diff
@@ -0,0 +1,49 @@
+// Type definitions for Mustache 0.8.2
```

Comment:
We don't need them, but they're a nice to have so if you want to develop on the client, you get completion in Code immediately. I'd be fine if we just removed them and added typings to .gitignore though.